client/tests/test.ts

```diff
@@ -1,6 +1,168 @@
-import { expect, test } from "@playwright/test";
+import { expect, test, type Frame, type Locator, type Page } from "@playwright/test";
 
-test("index page has expected h1", async ({ page }) => {
-	await page.goto("/");
-	await expect(page.getByRole("heading", { name: "Welcome to SvelteKit" })).toBeVisible();
+const TEST_URL = "https://example.com/test";
+
+type FormSubmit = {
+	address: string;
+	recaptcha: string;
+	parachain_id?: string;
+};
+
+const getFormElements = async (page: Page, getCaptcha = false) => {
+	let captcha: Locator = {} as Locator;
+	if (getCaptcha) {
+		// ?: Hack. We need to wait for the frame to load and then invade it.
+		await page.reload();
+		const captchaFrame = await new Promise<Frame>(function (resolve, reject) {
+			let i = 0;
+			// function that waits for the frame and timeouts after 3 seconds
+			(function waitForFrame() {
```

Comment:
`import { until } from "opstooling-js";`?

Reply:
I'm having a problem while importing the package. It is empty.
Have you encounter the same problem?

![image](https://user-images.githubusercontent.com/8524599/226683017-224bdd9e-3829-48fe-887c-6ca79443e67b.png)

Reply:
Thanks @rzadp, it was imported now.

But now I'm having problems with playwright importing it:
> Error: No "exports" main defined in /Users/asd/Repositories/substrate-matrix-faucet/client/node_modules/opstooling-js/package.json imported from /Users/ads/Repositories/substrate-matrix-faucet/client/tests/test.ts

To be honest I don't wanna try to fight the compiler to only use one function, is it ok if I just copy and paste the method?

Reply:
I guess, let's just leave it like that for now, and revisit it later.